test/tests/p5.Distortion.js

```diff
@@ -10,7 +10,7 @@ describe('p5.Distortion', function () {
     expect(distortion.waveShaperNode).to.have.property('context').to.not.be
       .null;
     //default case
-    expect(distortion.amount).to.equal(500);
+    expect(distortion.amount).to.equal(0.25);
```

Comment:
Do you think we should also keep assertions that the right curve value gets passed to the wave shaper node? e.g. keep this modification you made, but add an additional line like this for each test case:
```js
expect(distortion.waveShaperNode.curve).toEqual(500)
```

Reply:
yes, thanks for the comments, I'll be working on this

Reply:
I changed my mind, i think it shouldn't be tested.

Reply:
Ah yes I see what you mean, it would make more sense to unit test that `makeDistortionCurve` function separately, but that can be a separate endeavour.